From 6d05625ca1ec9d8f9379035a06a762e40eb18276 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 08 Oct 2018 09:45:12 +0300
Subject: [PATCH] goya-release: fix target

Make shell is not BASH by default.
Do not use pushd popd
---

diff --git a/Makefile b/Makefile
index 7752fa2..29dc6e2 100644
--- a/Makefile
+++ b/Makefile
@@ -94,12 +94,12 @@
 	$(error $(NO_RELEASE_BRANCH_ERROR))
 else
 	@for dir in $(RELEASE_REPOS); do \
-		pushd ../$$dir > /dev/null; \
+		cd ../$$dir; \
 		if [ $$(git branch -r --points-at HEAD | grep -cE "origin/$(RELEASE_BRANCH)") -lt 1 ]; then \
 			echo "$$dir - is not aligned with origin/$(RELEASE_BRANCH)"; \
 			ALL_REPOS_ALIGNED=false; \
 		fi; \
-		popd > /dev/null; \
+		cd - > /dev/null; \
 	done; \
 	$$ALL_REPOS_ALIGNED
 endif
